isthmus/src/main/java/io/substrait/isthmus/SqlToSubstrait.java

```diff
@@ -66,8 +67,10 @@ public Plan execute(String sql, List<String> tables) throws SqlParseException {
 
     if(tables != null) {
       for (String tableDef : tables) {
-        DefinedTable t = parseCreateTable(factory, validator, tableDef);
-        rootSchema.add(t.getName(), t);
+        List<DefinedTable> tList = parseCreateTable(factory, validator, tableDef);
```

Comment:
This was designed to support a user passing multiple create commands by passing same arg multiple times.

`cmd ... -- create "t1..." -- create "t2..." -- create "t1..."`

Does that not work? If not, let's fix it. It looks like here you're having user pass multiple CREATEs as a single string.

Reply:
The above syntax works.  Yes, the change proposed  is to allow user to pass a multiple statements block. For example, in tpch_smoke.sh, we could pass the whole contents in schema.sql, which contains 8 create table statements. Without the change, we have to separate schema.sql into 8 DDL strings, and use  
```
--create "create nation" --create "create orders" --create "create lineitem" .... 
```

The proposed change does not break existing usage. Use rnow could pass either single or multiple DDL statement in one --create string.

Reply:
It seems like an anti-pattern for a cli. Is there a reason you want to support all of the syntax in a single string? 

If you need this change, can you add a junit test case and smoke test for it as well?

Reply:
I feel it convenient to support multiple table DDL in one --create string.  In case that DDL comes from an existing file, it could read from the file directly without manually copy / separate the DDLs before user could use this utility. 

But I do not have strong opinion. I could manually do the DDL editing, and use the existing syntax of this utility.

Reply:
I don't have one really either. Seems like adding support for multiple creates in a single statement just makes things easier in certain cases. So maybe just update the help text for the cli to clarify this behavior and the test cases I mentioned?

Reply:
- updated the help text for the cli of PlanEntryPoint. 
- added a new Junit test case in SimplePlansTest
- added tpch_smoke.sh as smoke test.

Reply:
To clarify here, you're not executing the tpch smoke test as a precommit time test because it doesn't pass yet, right?

Reply:
tpch_smoke.sh only runs the 7 queries that are supported. It would pass if I run manually. 

I added this tpch_smoke.sh to pr.ymal under 'Smoke test', follow 'smoke.sh' script.  I'm not clear how to run this smoke test, except for running it manually.